integration/cilium_client_daemon_test.go

```diff
@@ -29,13 +30,32 @@ import (
 	cnd "github.com/cilium/cilium/daemon/daemon"
 	cns "github.com/cilium/cilium/daemon/server"
 
+	etcdAPI "github.com/coreos/etcd/clientv3"
+	consulAPI "github.com/hashicorp/consul/api"
 	. "gopkg.in/check.v1"
 )
 
 var (
-	EpAddr = net.IP{0xbe, 0xef, 0xbe, 0xef, 0xbe, 0xef, 0xbe, 0xef, 0xaa, 0xaa, 0xaa, 0xaa, 0x11, 0x11, 0x11, 0x12}
+	EpAddr    = net.IP{0xbe, 0xef, 0xbe, 0xef, 0xbe, 0xef, 0xbe, 0xef, 0xaa, 0xaa, 0xaa, 0xaa, 0x11, 0x11, 0x11, 0x12}
+	kvBackend = ""
 )
 
+func cfgKVBackend(c *cnd.Config) error {
```

Comment:
If this isn't used in production code, only on test code, it would fit nicely in a `testutil` package.